r/R/utilities.R

```diff
@@ -0,0 +1,29 @@
+#
+# subset_ids subsets a seurat object with the cell ids
+#
+subset_ids <- function(scdata, cells_id) {
+  meta_data_subset <- scdata@meta.data[match(cells_id, scdata@meta.data$cells_id), ]
+  current_cells <- rownames(meta_data_subset)
+  scdata <- subset(scdata, cells = current_cells)
+  return(scdata)
+}
+
+insertSetChildThroughApi <- function(new_cell_set, api_url, experiment_id, cell_set_key, auth_JWT) {
```

Comment:
I see two different naming conventions used here for naming functions - `insertSetChildThroughApi` with camel case and `subset_ids` with underscore, like you'd write python code. I don't know what is the standard in R. If there are no standards in R, I suggest you decide on which style you like and then you use on it consistently across the code. Same applies for variable names and also file names.

Reply:
subset_ids is wrong. I will change that. We already agreed that R functions will be camel case